### PR TITLE
fix #1095 changed modal layout  for both on startup and empty results

### LINKED ISSUE #1095: Avoid resize of the catalog when empty content
When search has empty results, or on startup the catalog appears too small. We should improve this, maybe using modal dialog that centers it automatically. 
Modal dialog means to use `<Dialog modal={true ...` . 
We should set up a fixed width (relative to the window) and responsive layout to view the items at different sizes depending on the size of the screen.

### PR DESCRIPTION
**Pull request by saidaipparla**
catalog

### CODE REVIEW

File: web/client/product/assets/css/viewer.css [hunk 1]
```diff
@@ -391,10 +391,7 @@ html, body, #container, .fill {
 
 #mapstore-catalog-panel {
     width: auto;
-    position: absolute;
-    right: 200px;
-    margin-top: 100px;
-    display: inline-block;
+    margin: 100PX;
```
Comment (offtherailz) [inline web/client/product/assets/css/viewer.css]: **not** capital px. The style should be responsive (if the width of the window is smaller, it should be 0, for instance).

File: web/client/plugins/MetadataExplorer.jsx [hunk 2]
```diff
@@ -55,6 +55,7 @@ const MetadataExplorerComponent = React.createClass({
             id: "mapstore-metadata-explorer",
             active: false,
             wrap: false,
+            modal: true,
```
Comment (offtherailz) [inline web/client/plugins/MetadataExplorer.jsx]: This have no effect, the modal parameter should be passed to the `Dialog` component.

**Comment by saidaipparla** [comment]
screenshots of final result
bigscreen
withresult
mapstore 2 homepage
screencapture-localhost-8081-1476175141360
screencapture-localhost-8081-1476175163910

**Comment by offtherailz** [comment]
@simboss What do you think? In my opinion we should put on the same line CSW/WMS sekectir and the URL (at least for larger screens)
Maybe we can make it smaller in wider screens.

**Pull request merged**